src/bridge/events.rs

```diff
@@ -148,7 +149,7 @@ pub enum RedrawEvent {
 
 fn unpack_color(packed_color: u64) -> Color4f {
     let packed_color = packed_color as u32;
-    let r = ((packed_color & 0xff0000) >> 16) as f32;
+    let r = ((packed_color & 0x00ff_0000) >> 16) as f32;
```

Comment:
nit: the underscore added here obscures the pattern of accessing the different hex values as color. I'd prefer it not be changed.

Reply:
Alternatively you could add an underscore between every pair of 2 hex values since thats how the colors are read. This isn't a number in the traditional sense

Reply:
I decided to keep the old format.